watson_developer_cloud/conversation_v1.py

```diff
@@ -89,21 +100,27 @@ def list_counterexamples(self,
                              cursor=None):
         """
         List counterexamples.
+
+        List the counterexamples for a workspace. Counterexamples are
+        examples that have been marked as irrelevant input.
+
         :param workspace_id: The workspace ID.
         :param page_limit: The number of records to return in each page of
             results. The default page limit is 100.
         :param include_count: Whether to include information about the number
             of records returned.
-        :param sort: The sort order that determines the behavior of the
-            pagination cursor.
+        :param sort: Sorts the response according to the value of the
+            specified property, in ascending or descending order.
         :param cursor: A token identifying the last value from the previous
             page of results.
         """
-        params = {'version': self.version}
-        params['page_limit'] = page_limit
-        params['include_count'] = include_count
-        params['sort'] = sort
-        params['cursor'] = cursor
+        params = {
+            'version': self.version,
+            'page_limit': page_limit,
+            'include_count': str(include_count).lower(),
```

Comment:
Great! I removed the code to cast boolean params to lowercase strings. I tested that with the examples and confirmed that the `_cleanup_param_values` was being called.

Should be ready to merge now. :)